kratos/sources/profiler.cpp

```diff
@@ -0,0 +1,285 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Máté Kelemen
+//
+
+// --- Core Includes ---
```

Comment:
Would you mind using the standard notation for includes?

Reply:
Sure. This started out as a copy-pasta from one of my pet projects, so there might still be some stuff that aren't in Kratos style. Have a look plz.